### DISCUSSION TITLE
Welcome to Grand-Ultimate-BTC-Hacker Discussions!

### DISCUSSION THREAD

**Discussion by room101-dev** [room101-dev/Grand-Ultimate-BTC-Hacker#7] [Announcements]
talking about explicit ways to hack bitcoin post ETH staking collapse of GPU mining

repurposing GPU cards to hacking BTC

**Comment by room101-dev** [comment]
besides running the hacking tool, you seed to harvest the new bitcoin addresses as they're created
 Every 4 hours get the new bitcoin addresses and add them to the bloom filter
#!/bin/bash

#lastblk=$(bitcoin-cli getblockcount)
# start-block, as nb, every 48 blocks flush found to blf, and update nb+48, about 12 hrs, takes 1hr with 4gb ram to update.

nb=$(echo $(bitcoin-cli getblockcount) "+48" | bc)

while [ 1 ]
do 

#sleep 10

blkcnt=$(bitcoin-cli getblockcount)

echo 'getblockcount', $blkcnt

# script to apply passwords to all new transactions on bitcoin
# python routine to get most recent block and harvest all addresses

nice -n 5 python get-newhex.py

sort -u base58.txt > addr.txt

# change addresses to base 58 form, make-h160 addr.txt->addr.hex
# input is base58.txt ( addr.txt ), output is addr.hex

python make-hash160.py

# update bin file

cat addr.txt >> address.txt
cat addr.hex >> address.hex

# generate bloom filter from base58 addresses 

echo 'blkcnt > nb', $blkcnt, $nb

if   [ "$blkcnt" -gt "$nb" ]; then
echo 'update bloom-filter', $nb
nice -n 5 sort -u -S 2G address.hex > address-uniq.hex
nice -n 5 hex2blf8 address-uniq.hex /mnt/Mining/Crypto/hex2blf/monster8.blf &
nb=$(echo $blkcnt "+48" | bc)
#cat address-uniq.hex >> /mnt/Mining/Crypto/address-uniq.hex
cat address-uniq.hex >> /media/robot/Temp-SSD/Crypto/new-block/address-uniq.hex
rm address.hex
rm address.txt
fi

# wait until next block available, 4M delay, on average the python step above takes 5 min, and 10min per btc block

newblk=$(bitcoin-cli getblockcount)
delay=1
echo $delay,$newblk
while [ "$delay" -gt "0" ];
do
    if [ "$blkcnt" -eq "$newblk" ]; then
        sleep 4m
        newblk=$(bitcoin-cli getblockcount)
        echo 'newblk/wait',$newblk
    else
        delay=0

    fi
    
    sleep 1s
done


done

**Reply by Princd** [reply]
hello can i have your contact so we can talk more i got a metter idea so let discuss this zou cann hit me on gmail 
monalisa.lee2018@gmail.com

**Comment by justusgeorge** [comment]
Can you guys help out if it’s a software or something I can get to hack bitcoin or crypto or nft or anything thing to yield stable income
